src/test/java/com/hocs/test/glue/decs/SearchStepDefs.java

```diff
@@ -240,10 +202,9 @@ public void createdCaseShouldBeVisibleInTheSearchResults(){
                 retest ++;
                 dashboard.selectSearchLinkFromMenuBar();
                 if (getCurrentCaseType().equalsIgnoreCase("MIN") || getCurrentCaseType().equalsIgnoreCase("TRO") || getCurrentCaseType().equalsIgnoreCase("DTEN")) {
-                    iEnterIntoTheSearchFieldForTheCaseType(getCurrentCaseReference(), "Case Reference", "DCU");
-
+                    iEnterIntoTheSearchFieldForTheCaseType(getCurrentCaseReference(), "Case Reference");
                 } else {
-                    iEnterIntoTheSearchFieldForTheCaseType(getCurrentCaseReference(), "Case Reference", getCurrentCaseType());
+                    iEnterIntoTheSearchFieldForTheCaseType(getCurrentCaseReference(), "Case Reference");
                 }
                 safeClickOn(searchButton);
                 workstacks.filterByCurrentCaseReference();
```

Comment:
in 2 places this method filters the workstack by a case reference before asserting on the total. This seems incorrect for the assertion we are making.